src/main/java/net/projectzombie/region_rotation/controller/BaseStateCommands.java

```diff
@@ -0,0 +1,144 @@
+package net.projectzombie.region_rotation.controller;
+
+import com.sk89q.worldguard.bukkit.WGBukkit;
+import net.projectzombie.region_rotation.file.FileRead;
+import net.projectzombie.region_rotation.modules.BaseState;
+import net.projectzombie.region_rotation.modules.StateController;
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+
+import static net.projectzombie.region_rotation.controller.BaseStateText.*;
+
+/**
+ * Location of all the commands for BaseState changes in-game.
+ * @author Gephery
+ */
+public class BaseStateCommands implements CommandExecutor
+{
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (args.length > 0)
+        {
+            if (args[0].equals(ADD_BASESTATE_CMD) && sender.hasPermission(ADD_BASESTATE_PERM))
+            {
+                boolean success = false;
+                BaseState baseState = null;
+                if (args.length == 5)
+                {
+                    // Convert all the inputs
+                    String regionName = args[1];
+                    World world = Bukkit.getWorld(args[2]);
+                    String backupRegionName = args[3];
+                    World backupWorld = Bukkit.getWorld(args[4]);
+
+                    if (world != null && backupWorld != null
```

Comment:
You could create the BaseState, then call `if (BaseState.isValid()) { StateController.instance().addBaseState }`